src/lib/features/feature-toggle/converters/feature-toggle-row-converter.ts

```diff
@@ -149,6 +151,14 @@ export class FeatureToggleRowConverter {
 
             feature = this.createBaseFeature(r, feature, featureQuery);
 
+            if (
+                this.isUnseenStrategyRow(feature, r) &&
+                includeDisabledStrategies &&
+                r.strategy_disabled
+            ) {
+                feature.strategies?.push(this.rowToStrategy(r));
+            }
+
```

Comment:
Why does the feature toggle list need disabled strategies?

Reply:
Following to the frontend - the disabled strategies are not needed